packages/react-components/react-virtualizer/src/components/Virtualizer/useVirtualizer.ts

```diff
@@ -276,23 +276,26 @@ export function useVirtualizer_unstable(props: VirtualizerProps): VirtualizerSta
     return childProgressiveSizes.current[numItems - 1] - childProgressiveSizes.current[lastItemIndex];
   };
 
-  const updateChildRows = (newIndex: number) => {
-    if (numItems === 0) {
-      /* Nothing to virtualize */
+  const updateChildRows = useCallback(
```

Comment:
now that you're memoizing `updateChildRows` can you add them to the dep array in the effect? otherwise I can't see why you would need to memoize this function https://github.com/microsoft/fluentui/blob/5df23bfe910893789b4e1dee016e8cc7d115d394/packages/react-components/react-virtualizer/src/components/Virtualizer/useVirtualizer.ts#L379-L393